build/lib/android/index.js

```diff
@@ -40,6 +43,18 @@ class Android {
 		};
 	}
 
+	babelOptions() {
+		const v8Version = require(path.join(ROOT_DIR, 'android', 'package.json')).v8.version;
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/android...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/android/index.js;
  Line: 47;
-->

- :warning: build/lib/android/index.js line 47 – Found non-literal argument in require (security/detect-non-literal-require)